cfgov/unprocessed/js/modules/util/expanded-state.js

```diff
@@ -63,12 +62,26 @@ function toggleExpandedState( $elem, state, cb, delay ) {
   }
 }
 
-module.exports = {
-  get: {
-    isThisExpanded: isThisExpanded,
-    isOneExpanded:  isOneExpanded
-  },
-  set: {
-    toggleExpandedState: toggleExpandedState
+/**
+ * Change the value of the element's aria-expanded attribute based on it's
+ * current state if one is not passed. Allows for a delayed event for
+ * animated elements.
+ * @param {NodeList} elems Element on which to set the aria-expanded attribute.
+ * @param {string} state The value, if any, to set the aria-expanded
+ *   attribute to. Options are "true", "false" or null.
+ * @param {Function} cb The callback function to execute after a delay.
+ * @param {integer} delay The length of time to delay the execution of
+ *                        the passed callback.
+ */
+function toggleGroupExpandedState( elems, state, cb, delay ) {
```

Comment:
So the problem is the `elems` parameter. It's a NodeList, so we'd either need to use querySelectorAll for the single elements, which doesn't seem so clear/efficient when there's only one element to retrieve, or convert the NodeList to an array, which is kind of ugly. E.g.:

```
toggleGroupExpandedState( [].slice.call( _primaryLinks ), 'false' );
```

What do you think?